<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="passwordInput">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="emailInput">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::Normal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="aceptatButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>370</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Registrarse</string>
    </property>
   </widget>
   <widget class="QLabel" name="loginTitle">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Registrarse</string>
    </property>
   </widget>
   <widget class="QLabel" name="emailLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLabel" name="passwordLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Contrasena</string>
    </property>
   </widget>
   <widget class="QLabel" name="passwordLabel_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>430</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: blue;</string>
    </property>
    <property name="text">
     <string>¿Ya tienes cuenta? Inicia sesión</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>emailInput</tabstop>
  <tabstop>passwordInput</tabstop>
  <tabstop>aceptatButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
